docs/src/declarative_database_management.md

```diff
@@ -1,54 +1,130 @@
-# Declarative Database Management
+# PostgreSQL Database Management
 
-Declarative database management enables users to control the lifecycle of
-databases via a new Custom Resource Definition (CRD) called `Database`.
+CloudNativePG simplifies PostgreSQL database provisioning by automatically
```

Comment:
I think this makes more sense if we speak of cluster.
```suggestion
CloudNativePG simplifies PostgreSQL cluster provisioning by automatically
```

Reply:
I actually meant the database.

Reply:
So, we say CNPG simplifies database provisioning by creating the `app` database automatically.
An the rest of the document talks about a completely different way of creating databases.

Reply:
The idea is to explain why we always create the app database.

Reply:
👍